plugins/split-yaml.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: split-yaml
+spec:
+  version: v0.0.1
+  homepage: https://github.com/nathforge/kubectl-split-yaml
+  shortDescription: Save Kubernetes YAML output into one file per resource.
+  description: This plugin takes output from `kubectl get -o yaml` and saves one file
+    per resource.
```

Comment:
We usually don't recommend usage instructions here, because most plugins run standalone. However, your plugin is designed to be piped into, so it would make sense to give an example here.

You could also mention that it works well with `get-all`, since this is explicitly mentioned on your landing page and `get-all` is also a krew plugin. I might have a conflict of interest here though, so it's completely up to you.

Reply:
Have done thanks. I've only added one example as `krew info` seems to collapse whitespace so wanted to keep it brief.